src/App.tsx

```diff
@@ -8,33 +9,18 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTab, setSelectedTab] = useState(tabs[0] || null);
+
   return (
-    <div className="section">
-      <h1 className="title">
-        Selected tab is&nbsp;Tab 1
+    <div className="App">
+      <h1 className="App__title">
+        {`Selected tab is ${selectedTab.title}`}
       </h1>
-
-      <div>
-        <div className="tabs is-boxed">
-          <ul>
-            <li className="is-active">
-              <a href="#tab-1">Tab 1</a>
-            </li>
-
-            <li className="">
-              <a href="#tab-2">Tab 2</a>
-            </li>
-
-            <li className="">
-              <a href="#tab-3">Tab 3</a>
-            </li>
-          </ul>
-        </div>
-
-        <div className="block" data-cy="tab-content">
-          Some text 1
-        </div>
-      </div>
+      <Tabs
+        tabs={tabs}
+        selectedTabId={selectedTab.id}
+        onTabSelected={setSelectedTab}
```

Comment:
what happens if you select the same tab? you'll perform select action anyway
make sure you review p.4 of the checklist